x/erc20/types/utils.go

```diff
@@ -35,3 +36,19 @@ func EqualMetadata(a, b banktypes.Metadata) error {
 	}
 	return fmt.Errorf("metadata provided is different from stored")
 }
+
+func EqualStringSlice(a, b []string) bool {
```

Comment:
can you add a code comment on top of this function?

Reply:
> can you add a code comment on top of this function?

Finished.